addons/web/static/src/webclient/actions/client_actions.js

```diff
@@ -14,6 +14,7 @@ export const displayNotificationAction = (env, action) => {
         sticky: params.sticky || false,
         title: params.title ? escape(params.title) : "",
         type: params.type || "info",
+        messageIsHtml: params.messageIsHtml || false,
     };
     let links = (params.links || []).map((link) => {
         return `<a href="${escape(link.url)}" target="_blank">${escape(link.label)}</a>`;
```

Comment:
`${escape(link.url)}` won't this turn the `action=42&id=12` into `action=42&amp;id=12`?